src/middleware/errorHandler.js

```diff
@@ -1,14 +1,13 @@
 const Sentry = require('@sentry/node');
-const express = require('@feathersjs/express');
 const { Configuration } = require('@hpi-schul-cloud/commons');
 const jwt = require('jsonwebtoken');
 const OpenApiValidator = require('express-openapi-validator');
 
-const reqlib = require('app-root-path').require;
-
-const { SilentError, PageNotFound, AutoLogout, BruteForcePrevention, BadRequest } = reqlib('src/errors');
-const { convertToFeathersError, cleanupIncomingMessage } = reqlib('src/errors/utils');
+const { SilentError, PageNotFound, AutoLogout, BruteForcePrevention } = require('../errors');
+const { ValidationError, DocumentNotFound, AssertionError, InternalServerError } = require('../errors');
+const { API_VALIDATION_ERROR } = require('../errors/commonErrorTypes');
```

Comment:
This is only the key base object not the real 'instance of' error. The naming is little bit confusing for me.

Reply:
done